core/workspace_comment/workspace_comment_render_svg.js

```diff
@@ -30,4 +30,134 @@ goog.require('Blockly.WorkspaceCommentSvg');
 
 Blockly.WorkspaceCommentSvg.prototype.render = function() {
   console.log('Rendering a comment!');
+
+  this.renderDraw_();
+};
+
+/**
+ * Draw the path of the block.
+ * Move the fields to the correct locations.
+ *     containing position information.
+ * @private
+ */
+Blockly.WorkspaceCommentSvg.prototype.renderDraw_ = function() {
+  var steps = [];
```

Comment:
Liens 44-56 can be a `setPath` function, and it doesn't need to be broken down into top, right, left, and bottom (since it's just making a box).

Reply:
Sounds good.